.changeset/great-eyes-stare.md

```diff
@@ -0,0 +1,5 @@
+---
+'@microsoft/atlas-site': patch
```

Comment:
Should this be minor since we're not fixing existing atomics?

Reply:
Updated!